pptx/oxml/shapes/groupshape.py

```diff
@@ -274,5 +280,24 @@ class CT_GroupShapeProperties(BaseOxmlElement):
         "a:extLst",
     )
     xfrm = ZeroOrOne("a:xfrm", successors=_tag_seq[1:])
+    eg_lineFillProperties = ZeroOrOneChoice(
```

Comment:
I'm curious why you named this lineFillProperties instead of grpFillProperties?

Reply:
Apparently, I copied that block from the Line section and forgot to rename. Fixed!